.github/workflows/JuliaNightly.yml

```diff
@@ -13,16 +13,14 @@ jobs:
         with:
           version: nightly
           arch: x64
-      - uses: actions/cache@v1
+      - uses: actions/cache@v2
         env:
-          cache-name: cache-artifacts
+          cache-name: julia-nightly-cache-artifacts
         with:
           path: ~/.julia/artifacts
-          key: ${{ runner.os }}-test-${{ env.cache-name }}-${{ hashFiles('**/Project.toml') }}
+          key: ${{ env.cache-name }}-${{ hashFiles('**/Project.toml') }}
```

Comment:
Just curious why we don't keep these the same between CI.yml and here?

Reply:
Hmm I might have incorrectly assumed since they were in completely separate files/pipelines that the caches wouldn't overlap, this I chose to minimize the key on this one since the `runner.os` is always the same for this test.

Reply:
I don't know that this confirms if caches work across all ci configs in a project specifically https://github.com/actions/cache but thinking about it more, it might just be project wide